docs/intro-to-coalton.md

```diff
@@ -410,14 +428,9 @@ The following type classes are similar to the typeclasses with the same name in
 * `Applicative`
 * `Monad` - monad does not have `return`, use `pure` from applicative instead
 * `Alternative` - `<|>` is called `alt` in Coalton
-
-
-
```

Comment:
Those were load bearing newlines. `Into` and `TryInto` aren't in Haskell. There should probably be an explanation that those two type classes are analogous  to the traits with the same names in Rust.